src/App.jsx

```diff
@@ -14,33 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const IS_SORT_BY_ALPHABET = 'alphabet';
+const IS_SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const getGoods = (goods, sortType, isReversed, isReseted) => {
+  let visibleGoods = [...goods];
```

Comment:
```suggestion
  const visibleGoods = [...goods];
```
`sort` and `reverse` mutate arrays so you can do it const